tests/framework/db/mssql/SchemaTest.php

```
@@ -208,4 +210,171 @@ public function testGetPrimaryKey(): void
+                '[server1].catalog1.sales.customer',
+                'catalog1',
+                'sales',
+                'customer',
+                'catalog1.sales.customer',
+            ],
+        ];
+    }
+
+    public function testSavepointOperations(): void
+    {
+        $db = $this->getConnection(true, true);
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)